kernel/src/sched.rs

```diff
@@ -602,7 +602,8 @@ impl Kernel {
             }
 
             // Check if the scheduler wishes to continue running this process.
-            if !scheduler.continue_process(process.appid(), chip) {
+            let continue_process = unsafe { scheduler.continue_process(process.appid(), chip) };
```

Comment:
This maybe should not be an unsafe function at all, to be consistent with the other changes here.

Reply:
Right, that should be a separate PR.